### ISSUE TITLE
Error on login

### ISSUE THREAD

**Issue by danielchicote** [magento-hackathon/AdvancedAcl#24]
Hello, i have installed the module and cleared the cache, but when i try to login into the backend i have this error.

Fatal error: Call to a member function getRole() on a non-object in C:\xampp\htdocs\magento\app\code\community\MagentoHackathon\AdvancedAcl\Helper\Data.php on line 152

Whats happened?

**Comment by sprankhub** [comment]
Did you clear the cache?

**Comment by danielchicote** [comment]
Yes, i think that there is problem with the BL Customgrid module in the Mage_Admin_Model_Role

**Comment by mayWz** [comment]
@danielchicote Hi, i have same problem. Did you fix this?
